recipes/pygsl_lite/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "pygsl_lite" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pygsl_lite-{{ version }}.tar.gz
+  sha256: 5f72ba936e49eb6dafd5fb78668e50538b8ef9d1e321d937018c5cad0b4227a4
+
+build:
+  skip: true  # [py<38 or win]
```

Comment:
Python <3.8 is not supported, so that selector can be removed:

```suggestion
  skip: true  # [win]
```

Reply:
thanks, removed